code/__DEFINES/anglegrinder.dm

```diff
@@ -0,0 +1,8 @@
+// Config, comments shits out to disable features
+
+#define ANGLEGRINDER_OUTPOST_PRICE 1000
+// #define ANGLEGRINDER_BROKEN_MACHINE_SALVAGE_TIME 2.5 SECONDS
+#define ANGLEGRINDER_OPEN_SAFE_TIME 90 SECONDS
+#define ANGLEGRINDER_OPEN_CLOSET_TIME 5 SECONDS
+#define ANGLEGRINDER_RAILING_TIME 1 SECONDS
+#define ANGLEGRINDER_MACHINE_FRAME_TIME 3 SECONDS
```

Comment:
These should be config entries not defines.

Reply:
Honestly I don't know how to do all of this cleanly using config without making some wacky code or hurting performance. I could have also hard coded all the value as it's usually done, but it would make balancing and tweaking of the feature harder.